src/xacro/__init__.py

```diff
@@ -950,7 +950,7 @@ def process_doc(doc,
     if do_check_order and symbols.redefined:
         warning("Document is incompatible to --inorder processing.")
         warning("The following properties were redefined after usage:")
-        for k, v in symbols.redefined.iteritems():
+        for k, v in symbols.redefined.items():
```

Comment:
Although iteritems() is more efficient than items() in python2, I think for the sake of simplicity, we can switch to items().